mtp_cashbook/apps/disbursements/tests/test_create.py

```diff
@@ -427,8 +427,8 @@ def test_account_details_required(self):
         self.enter_recipient_address()
         response = self.enter_recipient_bank_account({'sort_code': ''})
         self.assertOnPage(response, 'disbursements:recipient_bank_account')
-        self.assertFormError(response, 'form', 'sort_code', 'This field is required')
-        self.assertFormError(response, 'form', 'account_number', 'This field is required')
+        self.assertFormError(response, 'form', 'sort_code', 'This field is required.')
+        self.assertFormError(response, 'form', 'account_number', 'This field is required.')
```

Comment:
same issue as in noms-ops
https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/617#discussion_r2007484383

Reply:
FYI: Will be sorted by this + the `mtp-common` release: https://github.com/ministryofjustice/money-to-prisoners-cashbook/pull/614/commits/3e366c8c70180834f32bf96cfe244445b8668af2